analyzer.py

```diff
@@ -70,8 +70,11 @@ def calcScore(dic):
                 pass
 
         if categoryN == "reviews":
-            workloads.append([courseN, calcScore(sheet["1.6"])])
-            qualityscores.append([courseN, calcScore(sheet["1.8"])])
+            try:
```

Comment:
For at forbedre kvaliteten løbende må der meget gerne tilføjes flere kommentarer i koden